aviary/subsystems/propulsion/motor/motor_builder.py

```diff
@@ -0,0 +1,125 @@
+    build_pre_mission(self) -> openmdao.core.System:
+        Builds an OpenMDAO system for the pre-mission computations of the subsystem.
+    build_mission(self, num_nodes, aviary_inputs) -> openmdao.core.System:
+        Builds an OpenMDAO system for the mission computations of the subsystem.
+    get_constraints(self) -> dict:
+        Returns a dictionary of constraints for the motor subsystem, where the keys are the names of the variables to be constrained, and the values are dictionaries that contain the lower and upper bounds for the constraint and any additional keyword arguments accepted by Dymos for the constraint.
+    get_design_vars(self) -> dict:
+        Returns a dictionary of design variables for the motor subsystem, where the keys are the names of the design variables, and the values are dictionaries that contain the units for the design variable, the lower and upper bounds for the design variable, and any additional keyword arguments required by OpenMDAO for the design variable.
+    get_parameters(self) -> dict:
+        Returns a dictionary of fixed values for the motor subsystem, where the keys are the names of the fixed values, and the values are dictionaries that contain the fixed value for the variable, the units for the variable, and any additional keyword arguments required by OpenMDAO for the variable.
+    get_initial_guesses(self) -> dict:
+        Returns a dictionary of initial guesses for the motor subsystem, where the keys are the names of the initial guesses, and the values are dictionaries that contain the initial guess value, the type of variable (state or control), and any additional keyword arguments required by OpenMDAO for the variable.
+    get_mass_names(self) -> list:
+        Returns a list of names for the motor subsystem mass.
+    preprocess_inputs(self, aviary_inputs) -> aviary_inputs:
+        No preprocessing needed for the motor subsystem.
+    '''
+
+    def __init__(self, name='motor', include_constraints=True):
+        self.include_constraints = include_constraints
+        super().__init__(name)
+
+    def build_pre_mission(self, aviary_inputs):
+        return MotorPreMission(aviary_inputs=aviary_inputs)
+
+    def build_mission(self, num_nodes, aviary_inputs):
+        return MotorMission(num_nodes=num_nodes, aviary_inputs=aviary_inputs)
+
+    # def get_constraints(self):
+        # if self.include_constraints:
+        #     constraints = {
+        # Dynamic.Mission.Motor.TORQUE_CON: {
+        #     'upper': 0.0,
+        #     'type': 'path'
+        # }
+        # # TBD Gearbox torque constraint
+        #     }
+        # else:
+        #     constraints = {}
+
+        # return constraints
+
+    def get_design_vars(self):
+        DVs = {
+            Aircraft.Engine.SCALE_FACTOR: {
+                'units': 'unitless',
+                'lower': 0.001,
+                'upper': None
+            },
+            Aircraft.EngineGearbox.GEAR_RATIO: {
```

Comment:
I think we are missing a period here. Should this be `Aircraft.Engine.Gearbox.GEAR_RATIO`